src/navigation/__stories__/PageConstructor.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import {Meta, Story} from '@storybook/react/types-6-0';
+
+import {PageConstructor, PageConstructorProps} from '../../containers/PageConstructor';
+import {NAVIGATION} from '../../demo/constants';
+
+import data from './data.json';
+
+export default {
+    title: `${NAVIGATION}/PageConstructor`,
+    component: PageConstructor,
+} as Meta;
+
+const DefaultTemplate: Story<PageConstructorProps> = (args) => <PageConstructor {...args} />;
```

Comment:
If this story is about navigation, then maybe you need to show the navigation's props only, and make the rest of the page unchanged?

`const DefaultTemplate: Story<NavigatinProps> = (args) => <PageConstructor {..._.omit(data, 'navigation')} {...args} />;`


for example:
```
DefaultNavigation.args = data.navigation as NavigationProps;
```